ci/quickstart-jira-dc-params.json

```diff
@@ -42,5 +42,9 @@
     {
         "ParameterKey": "KeyPairName",
         "ParameterValue": "replaced-by-taskcat-override-file"
+    },
+    {
+        "ParameterKey": "BastionHostRequired",
```

Comment:
I wonder if there is some implication to the AWS CI pipeline but hopefully it should be just easier for them